### PR TITLE
Bug 2026802: copy api-request upperbound for cluster-monitoring-operator

### PR DESCRIPTION
**Pull request by sthaha**
This patch addresses the bug by increasing the api-request upperbound of
cluster-monitoring-operator by copying the values from `master` for only
those platforms where the upperbound is lower than in `master`.
I.E. This only updates the upperbound of AWS, Azure, and GCP and not
BareMetal, VSphere and OpenStack since the ones in release-4.9 is higher
than the ones in master branch.

Signed-off-by: Sunil Thaha <sthaha@redhat.com>

### CODE REVIEW

**Comment by sthaha** [comment]
@ingvagabund I see that you have worked extensively on setting the api-request limits. Could you please see if this looks good to you?

**Comment by ingvagabund** [comment]
@sthaha thank you for updating the upper bounds!!! The upper bounds set on the master branch are more realistic estimate. Unfortunately, the bounds are still changing so we might need another update in 4.9 eventually.

/approve
/lgtm

**Comment by sthaha** [comment]
@ingvagabund Thank you for the review, I think `/backport-risk-assessed` is a no-op 

isn't it `/label backport-risk-assessed` ?

**Review by soltysh — APPROVED** [review]
/approve
/label backport-risk-assessed

**Comment by ingvagabund** [comment]
@juzhao would you mind labeling the PR with `cherry-pick-approved` to complete the qa part?

File: test/extended/apiserver/api_requests.go [hunk 1]
```diff
@@ -115,7 +115,7 @@ var _ = g.Describe("[sig-arch][Late]", func() {
 				"cluster-autoscaler-operator":            44,
 				"cluster-baremetal-operator":             31,
 				"cluster-image-registry-operator":        119,
-				"cluster-monitoring-operator":            33,
+				"cluster-monitoring-operator":            124,
```
Comment (bparees) [inline test/extended/apiserver/api_requests.go:118]: this is a big increase.  is it well understood why the number of watches increased so much?

Comment (ingvagabund) [reply test/extended/apiserver/api_requests.go:118]: 4.9 threshold was computed by taking maximum from the last N jobs. 4.10 uses percentiles computed from jobs ran in the last 2 months. Still, both approaches are experimental. Unfortunately, there's currently no automation that would adjust the bounds periodically. So it is possible the 4.9 upper bound for CMO is more than enough.

**Pull request merged**